src/tests/SFA.DAS.EmployerIncentives.Functions.PaymentsProcess.AcceptanceTests/local.settings.json

```diff
@@ -0,0 +1,22 @@
+﻿{
```

Comment:
Should this file be excluded and added to the README, to prevent accidentally uploading and overwriting with local config?

Reply:
This is a file Rus added, it overwrites the application value one and contains no values. The DB_CONNECTION_STRING and LEARNER_MATCH_API_URL text is replaced at runtime. This is not the usual way we work but I think it's done for the func.exe to work correctly